Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -142,14 +144,8 @@ public virtual void Dispose()
 
         private static void DeclareAndBindErrorExchangeWithErrorQueue(IModel model, string exchangeName, string queueName, string routingKey)
         {
-            model.QueueDeclare(
-                queue: queueName,
-                durable: true,
-                exclusive: false,
-                autoDelete: false,
-                arguments: null);
-
-            model.ExchangeDeclare(exchangeName, ExchangeType.Direct, durable: true);
+            model.QueueDeclare(queueName, true, false, false, null);
```

Comment:
I think it became less readable than before.

Reply:
yeah adding the argument names might help